tests/migrations/2017_05_21_000000_create_accounting_ledgers_table.php

```diff
@@ -13,7 +13,7 @@ class CreateAccountingLedgersTable extends Migration
     public function up()
     {
         Schema::create('accounting_ledgers', function (Blueprint $table) {
-            $table->increments('id');
+            $table->uuid('id')->primary();
             $table->string('name');
             $table->enum('type', ['asset', 'liability', 'equity', 'income', 'expense']);
```

Comment:
enum is hard to maintain. keep the type control in the code. for example: policies or validation.